src/tokens.c

```diff
@@ -82,6 +82,7 @@ const tokenDefinition_t const TOKENS_ETHEREUM[NUM_TOKENS_ETHEREUM] = {
 	{{0x38,0xc8,0x7a,0xa8,0x9b,0x2b,0x8c,0xd9,0xb9,0x5b,0x73,0x6e,0x1f,0xa7,0xb6,0x12,0xea,0x97,0x21,0x69}, "AMO ", 18},
 	{{0x84,0x93,0x6c,0xF7,0x63,0x0A,0xA3,0xe2,0x7D,0xd9,0xAf,0xF9,0x68,0xb1,0x40,0xd5,0xAE,0xE4,0x9F,0x5a}, "AMTC ", 8},
 	{{0x96,0x0b,0x23,0x6A,0x07,0xcf,0x12,0x26,0x63,0xc4,0x30,0x33,0x50,0x60,0x9A,0x66,0xA7,0xB2,0x88,0xC0}, "ANT ", 18},
+	{{0xa1,0x17,0x00,0x00,0x00,0xf2,0x79,0xD8,0x1A,0x1D,0x3c,0xc7,0x54,0x30,0xfA,0xA0,0x17,0xFA,0x5A,0x1e}, "ANTv2 ", 18},
```

Comment:
Not sure how Ledger would feel about this, but perhaps we can name the current version as "ANT" and rename the old one as "ANTv1" or "OLD_ANT"?